src/core/auth/auth.controller.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Param,
+  Response,
+  InternalServerErrorException,
+  BadRequestException,
+  UnauthorizedException,
+  ForbiddenException,
+} from '@nestjs/common';
+import { AuthService } from './service/auth.service';
+import { LoginDto } from './dto/login.dto';
+import { RegisterDto } from './dto/register.dto';
+import { ApiOkResponse, ApiResponse, ApiTags } from '@nestjs/swagger';
+import * as express from 'express';
+import { ApiException } from '@nanogiants/nestjs-swagger-api-exception-decorator';
+import { SuccessfullyLogin, SuccessfullyRegister } from './dto/responses.dto';
+import { INTERNAL_SERVER_ERROR } from 'src/constants/messages/messagesConst';
+import { MessageDTO } from 'src/common/dto/response.dto';
+@ApiTags('auth')
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('register')
+  @ApiOkResponse({
+    description: 'Returns the created user',
+    type: SuccessfullyRegister,
```

Comment:
```suggestion
    type: SuccessfullyRegisterDTO,
```
Suffix DTO missing right?